src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -925,8 +925,10 @@ private void IsSelectedChanged(object sender, PropertyChangedEventArgs e)
         /// <param name="subset">Subset of nodes that should be used for the search instead of the complete set of nodes. This is a list of NodeSearchElement types</param>   
         internal IEnumerable<NodeSearchElementViewModel> Search(string search, IEnumerable<NodeSearchElement> subset = null)
```

Comment:
We are calling this search() in test mode, to avoid the lucene search in that process. But after this change, doesn't this call the lucene search() in test mode. We are not even indexing in test mode, so I wonder if search tests will pass.

Reply:
I've rollback the changes in this method so the tests can pass correctly.
commit: https://github.com/DynamoDS/Dynamo/pull/14169/commits/f432e2c6fdaa272c1c10a4f9ebf532cbe397944c